GCC 8 Doesn't have support for -mtune=tsv110 yet (added in GCC 9.1), so use the similar cortex-a57 instead

author: Alexander Grund (TU Dresden)
diff --git a/Makefile.arm64 b/Makefile.arm64
index 4d10ff68..742041d6 100644
--- a/Makefile.arm64
+++ b/Makefile.arm64
@@ -40,6 +40,6 @@ FCOMMON_OPT += -march=armv8.1-a -mtune=thunderx2t99
 endif
 
 ifeq ($(CORE), TSV110)
-CCOMMON_OPT += -march=armv8.2-a -mtune=tsv110
-FCOMMON_OPT += -march=armv8.2-a -mtune=tsv110
+CCOMMON_OPT += -march=armv8.2-a -mtune=cortex-a57
+FCOMMON_OPT += -march=armv8.2-a -mtune=cortex-a57
 endif
